blockonomics.php

```diff
@@ -33,7 +33,7 @@ public function __construct()
     {
         $this->name = 'blockonomics';
         $this->tab = 'payments_gateways';
-        $this->version = '1.7.9';
+        $this->version = '1.8.0';
```

Comment:
We have used the Blockonomics module version to indicate PrestaShop compatibility in the past. eg v1.5 is compatible with PS 1.5 and v1.7 indicates compatibility with PS version 1.7. To keep in line with this standard I propose we use version 1.7.9.1, 1.7.9.2, 1.7.9.3, etc until the release of Prestashop 1.8, in case this version also creates breaking changes.

Reply:
Sounds good, I've changed the version number to 1.7.9.1